### PR TITLE
PMM-4912 Documentation for contributors

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-470/

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contributing notes
+
+## Pre-requirements: 
+git, make, curl, go, gcc, nginx, mkcert
```
Comment (AlekSi) [inline .github/CONTRIBUTING.md]: Why we need gcc?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Local setup
+Run `make -C api init` to install dependencies.
+
+#### To run nginx 
+Install latest nginx https://www.linuxbabe.com/ubuntu/install-nginx-latest-version-ubuntu-18-04
```
Comment (AlekSi) [inline .github/CONTRIBUTING.md]: That's too Ubuntu-specific. "Install latest nginx" is good enough.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-470/

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (BupycHuk) [reply .github/CONTRIBUTING.md]: Fixed

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (BupycHuk) [reply .github/CONTRIBUTING.md]: I started setting up from pmm-admin and it required gcc, so I copy-pasted it to all repos

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-470/

**Pull request merged**